.env

```diff
@@ -26,11 +26,17 @@ EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC=0
 # option, which could potential exclude all nodes from load balancer at the same time.
 MAX_CONCURRENT_UPDATE=1
 
-# Update time window setting
-# Brupop will operate node updates within update time window.
-# when you set up time window start and stop time, you should use UTC (24-hour time notation).
-UPDATE_WINDOW_START=0:0:0
```

Comment:
I think this is a good idea. While the `.env` file is a build-time artifact, these correlate to environment variables in the deployment, we should probably support the old-style window configuration so as to not break customers updating from `1.0.0` (but make it mutually exclusive with cron expressions).

Implementation wise, we could maybe convert the window into a cron expression and then use the same scheduler?

Reply:
Sorry for the very late review on this.

I don't think it makes sense anymore to continuing to maintaining an `.env` file that drops values into generated yaml once #350 merges.

If we plan to use helm templates for our yaml files (in https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/350) and _also_ use a `.env` build time file, then we are stuck with figuring out a way to interpolate _both_ a `values.yaml` file (which helm knows how to drop into its templates via `helm install -f values.yaml`) _and_ an `.env` file with it's own values. How would we reconcile the two if they clashed and tried to set the same value? How would we account for drift between the two?

In my opinion, once #350 merges, we should abandon trying to generate yaml from our rust code that ends up landing on clusters and commit to using `helm`.

Reply:
> I don't think it makes sense anymore to continuing to maintaining an .env file that drops values into generated yaml once https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/350 merges.

In general, we are just using `.env` as a way to configure the build process during development. If we still have cases where that makes sense, we should keep it. It's definitely not meant to be a place to control behavior on users' clusters though.

Reply:
Hmmmm, I think i'm having trouble seeing how the `.env` file is useful at buildtime and during development beyond dropping values into the generated yaml. I would argue that a developer should keep a `values.yaml` file that gets passed to `helm install -f`. 

But, I think we can table this part of the discussion; I'd like to get this PR in before the helm one so that there's a clean refactor & rebase there without this one having to adopt that new configuration model.